docs/workflow_solutions/using_anaconda.md

```diff
@@ -282,6 +282,94 @@ Now we can be sure that the correct versions of the software will be installed o
+
+    Then follow the installation instructions you see in the image below. ![!Anaconda.org page showing package installation instructions](images/install_anaconda_package.png)
+
+1. Using Conda Search Command: You can use the `conda search <package_name` command directly in your terminal to find packages. Replace `package_name` with the package you would like to search for. To do this on Cheaha, make sure to `module load Anaconda3` first, and follow the instructions to [activate](#activate-an-environment) an environment. Then do `conda search numpy`. You should get a long list of numpy packages. Review this output, but take note of the highlighted portions in the image. The section with a red selection shows the numpy versions that are available, The section with a blue selection shows the channel where each numpy version is stored. ![!Search output from using conda search in Terminal](images/channel_conda_search.png)
+
+    You can then install numpy with a specific version and from a specific channel with.
+
+    ```bash
+    conda install -c conda-forge numpy=2.0.0rc2
+
+    ```
+
+1. Using Specific Channels: You can also get packages using specific Anaconda Channels listed below.
+
+    1. Anaconda Main Channel: The default channel provided by Anaconda, Inc. Visit [Anaconda](https://anaconda.org)
+
+    1. Conda-Forge: A community-driven channel with a wide variety of packages.Visit [Conda-Forge](https://conda-forge.org/)
+
+    1. Bioconda: A channel specifically for bioinformatics packages. Visit [Bioconda](https://bioconda.github.io/)
+
+    You can specify a channel in your search, and it will show you a list of the packages available in that channel using `conda search -c <channel_name> <package_name>`, remember to replace <channel_name> and <package_name> with the channel and package names you are searching for respectively.
+
+    ```bash
+    conda search -c conda-forge numpy
+    ```
+
+If we find the package at one of these sources, we check the Platform version to ensure it is either noarch (if available) or linux. Noting the version, we can click the "source" or "repo" link (if available) or "homepage". Then we try to find the latest version. For a package found on GitHub, click "Releases" on the right-hand side. Verify that the latest Release is the same as, or very close to, the version on Anaconda or Pypi. If so, the package is being maintained on Anaconda/Pypi and suitable for use. Note the exact software name, version, and channel (if not on Pypi). ![!Github page for numpy, an Anaconda package](images/github_conda_releases.png)
```

Comment:
"either noarch (if available) or linux" for it to work on Cheaha. 

I'd make a note that "noarch" is preferred for portability to other systems, but isn't required for use on Cheaha.

Reply:
In addition to "source", "repo" and "homepage", there is also the "Development" link on Anaconda Cloud.